_episodes_rmd/05-dplyr.Rmd

```diff
@@ -88,6 +82,16 @@ memory. The database connections essentially remove that limitation in that you
 can have a database that is over 100s of GB, conduct queries on it directly and pull
 back just what you need for analysis in R.
 
+### Loading .csv files in tidy style
+
+The tidyverse provides its own unique way of loading .csv files in to R using `read_csv()`, which is similar to `read.csv()`. `read_csv()` allows users to load in their data faster, use tibbles instead of data.frames, doesn't create row names, and allows you to access non-standard variable names (ie. variables that start with numbers of contain spaces). In short, it's a much friendlier way of loading in potentially messy data.
```

Comment:
-> "The tidyverse (via the `readr` package) provides..."

Reply:
I would take out mention of tibbles, too much for this point.

Reply:
addressed in new commit of this pull request